### DISCUSSION TITLE
Unable to login after fresh install

### DISCUSSION THREAD

**Discussion by SebWo87** [nextcloud/all-in-one#3370] [Questions] [answered]
### Steps to reproduce
1. Installed docker from distribution repository
2. pulled AIO image and run with: `sudo docker run --init --sig-proxy=false --name nextcloud-aio-mastercontainer --restart always --publish 80:80 --publish 8080:8080 --publish 8443:8443 --volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config --volume /var/run/docker.sock:/var/run/docker.sock:ro --env NEXTCLOUD_DATADIR="/mnt/data/nextcloud" nextcloud/all-in-one:latest`
3. configured instance using the AIO interface
4. When trying to login with the initial password for user admin, nothing happens. When entering the password for many times, I get this message: "Es wurden mehrere ungültige Anmeldeversuche von deiner IP-Adresse festgestellt. Daher wird die nächste Anmeldung um 30 Sekunden verzögert."
5. Tried to reset the password with: `sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ user:resetpassword admin`. The command completed successfully but still unable to login.
6. Tried to reset the instance following [this](https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance) guide, but did not changed anything.

### Expected behavior 
Login with the user admin and the initial password works and I can access Nextcloud.
### Actual behavior 
Nothing happens when entering the password.

### Host OS 
Arch Linux
Linux hw-svr-01 6.4.11-hardened1-1-hardened #1 SMP PREEMPT_DYNAMIC Tue, 22 Aug 2023 19:24:19 +0000 x86_64 GNU/Linux

#### Nextcloud AIO version 
7.2.1 (tried before already with 7.0 with the same result)

#### Current channel 
latest
#### Other valuable info 
Nextcloud container first start (and restart) log:
`Waiting for database to start...
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
Installing imagemagick via apk...
Enabling Imagick...
Configuring Redis as session handler...
Setting php max children...
Initializing nextcloud 27.1.0.7 ...
Initializing finished
New Nextcloud instance.
Installing with PostgreSQL database
Starting Nextcloud installation...
Nextcloud was successfully installed
 - Repair MySQL collation
     - Not a mysql database -> nothing to do
 - Repair mime types
 - Clean tags and favorites
     - 0 tags of deleted users have been removed.
     - 0 tags for delete files have been removed.
     - 0 tag entries for deleted tags have been removed.
     - 0 tags with no entries have been removed.
 - Repair invalid shares
 - Move .step file of updater to backup location
 - Add move avatar background job
     - Add background job
 - Add preview cleanup background jobs
 - Migrate oauth2_clients table to nextcloud schema
     - Update the oauth2_access_tokens table schema.
     - Update the oauth2_clients table schema.
     - Delete clients (and their related access tokens) with the redirect_uri starting with oc:// or ending with *
 - Fix potential broken mount points
     - No mounts updated
 - Repair language codes
 - Add log rotate job
 - Clear frontend caches
     - Image cache cleared
     - JS cache cleared
 - Clear every generated avatar
 - Add preview background cleanup job
 - Queue a one-time job to cleanup old backups of the updater
 - Cleanup invalid photocache files for carddav
 - Add background job to cleanup login flow v2 tokens
 - Remove potentially over exposing share links
     - No need to remove link shares.
 - Clear access cache of projects
 - Reset generated avatar flag
 - Keep legacy encryption enabled
 - Check encryption key format
 - Remove old dashboard app config data
 - Add job to cleanup the bruteforce entries
 - Queue a one-time job to check for user uploaded certificates
 - Repair DAV shares
 - Add background job to set the lookup server share state for users
 - Add token cleanup job
 - Clean up abandoned apps
 - Add possibly missing system config
 - Add language model tasks cleanup job
 - Upgrading Circles App
 - Fix component of birthday calendars
     - 0 birthday calendars updated.
 - Regenerating birthday calendars to use new icons and fix old birthday events without year
     - Adding background jobs to regenerate birthday calendar
 - Fix broken values of calendar objects
    0 [>---------------------------]
 - Registering building of calendar search index as background job
     - Add background job
 - Register building of social profile search index as background job
 - Registering background jobs to update cache for webcal calendars
     - Added 0 background jobs to update webcal calendars
 - Registering building of calendar reminder index as background job
     - Add background job
 - Clean up orphan event and contact data
     - 0 events without a calendar have been cleaned up
     - 0 properties without an events have been cleaned up
     - 0 changes without a calendar have been cleaned up
     - 0 cached events without a calendar subscription have been cleaned up
     - 0 changes without a calendar subscription have been cleaned up
     - 0 contacts without an addressbook have been cleaned up
     - 0 properties without a contact have been cleaned up
     - 0 changes without an addressbook have been cleaned up
 - Remove activity entries of private events
     - Removed 0 activity entries
 - Clean up old calendar subscriptions from deleted users that were not cleaned-up
    0 [->--------------------------]
     - 0 calendar subscriptions without an user have been cleaned up
 - Remove invalid object properties
     - 0 invalid object properties removed.
 - Fix the share type of guest shares when migrating from ownCloud
 - Copy the share password into the dedicated column
 - Set existing shares as accepted
 - Update OAuth token expiration times
 - Switches from default updater server to the customer one if a valid subscription is available
 - Send an admin notification if monthly report is disabled
 - Force-reset all Text sessions before Yjs migration
 - Initialize migration of background images from dashboard to theming app
 - Add background job to check for backup codes
 - Populating added database structures for workflows
Applying default settings...
System config value loglevel set to string 2
System config value log_type set to string file
System config value logfile set to string /var/www/html/data/nextcloud.log
System config value log_rotate_size set to string 10485760
admin_audit 1.17.0 enabled
Config value logfile for app admin_audit set to /var/www/html/data/audit.log
System config value log.condition => apps => 0 set to string admin_audit
Applying preview settings...
System config value preview_max_x set to string 2048
System config value preview_max_y set to string 2048
System config value jpeg_quality set to string 60
Config value jpeg_quality for app preview set to 60
System config value enabledPreviewProviders deleted
System config value enabledPreviewProviders => 1 set to string OC\Preview\Image
System config value enabledPreviewProviders => 2 set to string OC\Preview\MarkDown
System config value enabledPreviewProviders => 3 set to string OC\Preview\MP3
System config value enabledPreviewProviders => 4 set to string OC\Preview\TXT
System config value enabledPreviewProviders => 5 set to string OC\Preview\OpenDocument
System config value enabledPreviewProviders => 6 set to string OC\Preview\Movie
System config value enabledPreviewProviders => 7 set to string OC\Preview\Krita
System config value enable_previews set to boolean true
Applying other settings...
System config value upgrade.disable-web set to boolean true
System config value mail_smtpmode set to string smtp
System config value trashbin_retention_obligation set to string auto, 30
System config value versions_retention_obligation set to string auto, 30
System config value activity_expire_days set to string 30
System config value simpleSignUpLink.shown set to boolean false
System config value share_folder set to string /Shared
deck 1.11.0 installed

bled
twofactor_totp 9.0.0 enabled
tasks 0.15.0 installed
tasks enabled
calendar 4.5.0 installed
calendar enabled
contacts 5.4.0 installed
contacts enabled
notes 4.8.1 installed
notes enabled
Applying one-click-instance settings...
System config value one-click-instance set to boolean true
System config value one-click-instance.user-limit set to integer 100
System config value one-click-instance.link set to string https://nextcloud.com/all-in-one/
support already enabled
Adjusting log files...
System config value upgrade.cli-upgrade-link set to string https://github.com/nextcloud/all-in-one/discussions/2726
System config value logfile set to string /var/www/html/data/nextcloud.log
Config value logfile for app admin_audit set to /var/www/html/data/audit.log
System config value updatedirectory set to string /nc-updater
Applying network settings...
System config value davstorage.request_timeout set to integer 3600
System config value trusted_domains => 1 set to string <domain>
System config value overwrite.cli.url set to string https://<domain>/
System config value htaccess.RewriteBase set to string /
.htaccess has been updated
System config value dbpersistent set to boolean true
System config value files_external_allow_create_new_local set to boolean false
nextcloud-aio 0.4.0 enabled
notify_push 0.6.3 installed
notify_push enabled
System config value trusted_proxies => 0 set to string 127.0.0.1
System config value trusted_proxies => 1 set to string ::1
Config value base_endpoint for app notify_push set to https://<domain>/push
richdocuments 8.2.0 installed
richdocuments enabled
Config value wopi_url for app richdocuments set to https://<domain>/
System config value allow_local_remote_servers set to boolean true
No ipv6-address found for <domain>.
Config value wopi_allowlist for app richdocuments set to 84.169.177.128,127.0.0.1/8,192.168.0.0/16,172.16.0.0/12,10.0.0.0/8,fd00::/8,::1
spreed 17.1.0 installed
spreed enabled
Added <domain>:3478.
Added <domain>:3478.
There is nothing to delete.
Added signaling server https://<domain>/standalone-signaling/.
Config value recording_servers of app spreed deleted
System config value enabledPreviewProviders => 0 set to string OC\Preview\Imaginary
System config value preview_imaginary_url set to string http://nextcloud-aio-imaginary:9000
[17-Sep-2023 20:00:33] NOTICE: fpm is running, pid 700
[17-Sep-2023 20:00:33] NOTICE: ready to handle connections
Activating Collabora config...
Activated any config changes
`

**Comment by szaimen** [comment, marked as answer]
Hi, I cannot reproduce this on my test server unfortunately. Did you already try if the login works from a different network and or browser?

**Reply by SebWo87** [reply]
Thanks a lot! This did the trick! Looks like an odd caching problem. I have reused the domain from a former Nextcloud instance so there might be some cookies or cached files causing trouble. A different browser works as well as Firefox (which I was using before) in private mode.

**Comment by reynold-lariza** [comment]
I am able to replicate this. I did try the same command to reset the admin password.

`nextcloud sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ user:resetpassword admin
Enter a new password:
Confirm the new password:
Successfully reset password for admin`

but it still says `The password is false`

**Comment by ghost** [comment]


**Reply by szaimen** [reply]
> I'm in some kind of loop where the "open next cloud" button takes me right back to the "containers" page. what happened to the regular login. How do I stop this auto-login crap.

Are you running AIO behind a reverse proxy?

**Reply by ghost** [reply]
it's a promox vm with cloudflare tunnel.  i'm looking for an older version.   the install was more straightforward then.  i'm recreated the vm and all...

**Reply by szaimen** [reply]
Please follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#6-how-to-debug-things then
